### DISCUSSION TITLE
Can't find the .env.local

### DISCUSSION THREAD

**Discussion by clintsanchez** [wpengine/faustjs#394] [Q&A] [answered]
Forgive me as im totally new.   I am trying to follow the tutorial but can't find the .env.local to add the live url.

**Comment by helmutgranda** [comment]
Their documentation is not in sync with the code since they still have the name of the file as .env.local.sample:

https://github.com/wpengine/faustjs/blob/canary/examples/next/getting-started/.env.local.sample

However, there is a PR to correct that.

**Comment by josephfusco** [comment, marked as answer]
Hey @clintsanchez, thanks for the feedback. The documentation is accurate now. Let us know if you have any other questions!
